src/Select/Select.js

```diff
@@ -94,6 +94,7 @@ function Select(props) {
       open,
       renderValue,
       SelectDisplayProps,
+      type: null, // We render a select. We can ignore the type provided by the `Input`.
```

Comment:
why didn't you use `undefined`, allows to do `type = 'hidden'` in SelectInput props destructuring

Reply:
It would trigger the Input defaultTypes.